docs/card-rendering.md

```diff
@@ -0,0 +1,20 @@
+# Card Rendering
+
+To render a card, the primary step is to obtain a Glimmer component identified by the template tags within the card. A card may contain several Glimmer components, including fields and other miscellaneous components, that are categorized into three formats: isolated, embedded, and edit. These formats determine which component should be rendered. If there are no components available for the isolated or edit formats, the [default format](../packages/base/default-card-component.gts) is used instead. The Glimmer components themselves are building blocks that can be used to create many other things.
```

Comment:
Is it true about the default format, or are templates inherited along the adoption chain? I assumed the latter. /cc @habdelra